client/grpc/oracle/proof/abi.go

```diff
@@ -26,7 +26,7 @@ var relayFormat = []byte(`
       },
       {
         "internalType": "bytes32",
-        "name": "tssToUpgradeStoresMerkleHash",
+        "name": "TSSToUpgradeStoresMerkleHash",
```

Comment:
This should be tss?

Reply:
if use tss, it will get an error "field not found"